k8s.io/README.md

```diff
@@ -20,14 +20,13 @@ Vanity URL(s)
 | Features repo | https://feature.k8s.io <br> https://features.k8s.io |  https://feature.kubernetes.io <br> https://features.kubernetes.io |
 | Install script | https://get.k8s.io | https://get.kubernetes.io |
 | Github organization| https://git.k8s.io | https://git.kubernetes.io |
-| Redirections | https://go.k8s.io | https://go.kubernetes.io |
```

Comment:
Why is this deleted?

Reply:
As far as I can see here it is an invalid url. it returns 404 not found.

Reply:
It's live. The config for it is here: https://github.com/kubernetes/k8s.io/blob/a7985e6b571de579941b2a78135a3035f7951d92/k8s.io/configmap-nginx.yaml#L246-L269

It's a redirector, so the / URL doesn't do anything.

Reply:
Got it, thanks. I will make a commit including all the required changes.